.github/workflows/publish-nuget.yml

```diff
@@ -7,6 +7,10 @@ on:
 
 jobs:
   call-publish-workflow:
-    uses: Lombiq/GitHub-Actions/.github/workflows/publish.yml@dev
+    uses: Lombiq/GitHub-Actions/.github/workflows/publish-nuget.yml@issue/OSOE-150
+    with:
+      # These warnings are related to the tooling erroneously assuming that a ps1 file must be an 
+      # (un)install script. In this repo the ps1 files are assets to be used by the consumer.
+      dotnet-pack-ignore-warning: NU5110 NU5111
```

Comment:
What do you mean by Build Action?

Reply:
No. As the comment above explains, the "problem" is that you have a PS script in the nuget package. But we need to pack the _Invoke-Analyzer.ps1_, that's where the code is.

If you have a ps1 file outside of the tools folder you get [NU5110](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5110) and if the file is not called init.ps1, install.ps1 or uninstall.ps1 then you get [NU5111](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5111). There is nothing we can or should do about that except disable the warnings because they are short-sighted and overbearing.

Reply:
I see.